backend/src/routes/virtualMachineProxy.ts

```diff
@@ -63,32 +62,17 @@ export async function virtualMachineProxy(req: Http2ServerRequest, res: Http2Ser
               return undefined
             })
 
-          // Get cluster proxy host
-          const proxyServer = await jsonRequest(
-            process.env.CLUSTER_API_URL +
-              '/apis/route.openshift.io/v1/namespaces/multicluster-engine/routes/cluster-proxy-addon-user',
-            token
-          )
-            .then((response: Route) => {
-              const scheme = response?.spec?.tls?.termination ? 'https' : 'http'
-              return response?.spec?.host ? `${scheme}://${response.spec.host}` : ''
-            })
-            .catch((err: Error): undefined => {
-              logger.error({ msg: 'Error getting cluster proxy Route', error: err.message })
-              return undefined
-            })
-
           // req.url is one of: /virtualmachines/<action> OR /virtualmachineinstances/<action>
           // the VM name is needed between the kind and action for the correct api url.
           const splitURL = req.url.split('/')
           const joinedURL = `${splitURL[1]}/${body.vmName}/${splitURL[2]}`
-          const path = `${proxyServer}/${body.managedCluster}/apis/subresources.kubevirt.io/v1/namespaces/${body.vmNamespace}/${joinedURL}`
+          const proxyService = 'https://cluster-proxy-addon-user.multicluster-engine.svc.cluster.local:9092'
```

Comment:
Namespace can be customized by customers, and should not be hardcoded. See https://github.com/stolostron/console/blob/main/backend/src/routes/aggregators/utils.ts#L100 for example.

The other parts of the service addressed being hardcoded is OK, as they won't change in a given released version.